schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0

```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an event where a mobile application is installed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
```

Comment:
I'm not sure if I like or not this overly generic semantics. It is not related to `com.snowplowanalytics.`, but at the same time there's no other vendor that can match such a generic event.

Reply:
@yalisassoon any thoughts?

Reply:
Let's keep it `com.snowplowanalytics.mobile`